src/lexedata/importer/interleaved.py

```diff
@@ -25,7 +31,7 @@
             except AttributeError:
                 break
 
-    w = csv.writer(open("forms.csv", "w"))
+    w = csv.writer(open(Path(forms_path) / "forms.csv", "w"))
```

Comment:
Return the list of orms instead, and let the wrapper script deal with the writing